bindings/matrix-sdk-crypto-ffi/src/machine.rs

```diff
@@ -800,6 +800,44 @@ impl OlmMachine {
         Ok(serde_json::to_string(&encrypted_content)?)
     }
 
+    /// Encrypt the given event with the given type and content for the given
+    /// device. This method is used to send an event to a specific device.
+    ///
+    /// # Arguments
+    ///
+    /// * `user_id` - The ID of the user who owns the target device.
+    /// * `device_id` - The ID of the device to which the message will be sent.
+    /// * `event_type` - The event type.
+    /// * `content` - The serialized content of the event.
+    ///
+    /// # Returns
+    /// A `Result` containing the request to be sent out if the encryption was
+    /// successful.
+    ///
+    /// The caller should ensure that there is an olm session (see
+    /// `get_missing_sessions`) with the target device before calling this
+    /// method.
+    pub fn encrypt_to_device_event(
```

Comment:
Should the name match the inner name? `create_encrypted_to_device_request`? The changelog suggests it should I think.